examples/linear_model/plot_poisson_regression_non_normal_loss.py

```diff
@@ -292,21 +315,40 @@ def score_estimator(estimator, df_test):
 
 ##############################################################################
 # The experimental data presents a long tail distribution for ``y``. In all
-# models we predict a mean expected value, so we will have necessarily fewer
-# extreme values. Additionally, the normal distribution used in ``Ridge`` and
-# ``RandomForestRegressor`` has a constant variance, while for the Poisson
-# distribution used in ``PoissonRegressor``, the variance is proportional to
-# the mean predicted value.
+# models, we predict the expected frequency of a random variable, so we will
+# have necessarily fewer extreme values than for the observed realizations of
+# that random variable. Additionally, the normal distribution used in ``Ridge``
```

Comment:
Not sure if relevant / clear but this is what I understand from the previous sentence:

```suggestion
# that random variable. This explains that the mode of the histogram doesn't correspond to the left-most value. Additionally, the normal distribution used in ``Ridge``
```

Reply:
I do not understand what "left-most value" refers to.

Reply:
I mean smallest

Reply:
Alright makes sense now. I will update.